planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -1166,8 +1200,7 @@ AvoidLineArray AvoidanceModule::extractShiftLinesFromLine(ShiftLineData & shift_
     // If the vehicle is already on the avoidance (checked by the first point has shift),
     // set a start point at the first path point.
     if (!found_first_start && std::abs(shift) > IS_ALREADY_SHIFTING_THR) {
-      utils::avoidance::setStartData(
-        al, 0.0, p, i, arcs.at(i));  // start length is overwritten later.
+      setStartData(al, 0.0, p, i, arcs.at(i));  // start length is overwritten later.
       found_first_start = true;
```

Comment:
`found_first_start` is whether start point of avoid line is found or not, correct?

Reply:
You're right!